src/openepd/api/epd/sync_api.py

```diff
@@ -103,3 +108,84 @@ def get_statistics(self, omf: str) -> StatisticsDto:
         :return: statistics wrapped in OpenEpdApiResponse
         """
         return self.get_statistics_raw(omf).payload
+
+    @overload
+    def post_with_refs(self, epd: Epd, with_response: Literal[True]) -> tuple[Epd, Response]:
+        ...
+
+    @overload
+    def post_with_refs(self, epd: Epd, with_response: Literal[False] = False) -> Epd:
+        ...
+
+    def post_with_refs(self, epd: Epd, with_response: bool = False) -> Epd | tuple[Epd, Response]:
+        """
+        Post an EPD with references.
+
+        :param epd: EPD
+        :param with_response: return the response object togather with the EPD
+        :return: EPD or EPD with HTTP Response object depending on parameter
+        """
+        response = self._client.do_request(
+            "patch",
+            "/epds/post-with-refs",
+            json=epd.to_serializable(exclude_unset=True, exclude_defaults=True, by_alias=True),
+        )
+        content = response.json()
+        if with_response:
+            return Epd.parse_obj(content), response
+        return Epd.parse_obj(content)
+
+    @overload
+    def create(self, epd: Epd, with_response: Literal[True]) -> tuple[Epd, Response]:
+        ...
+
+    @overload
+    def create(self, epd: Epd, with_response: Literal[False] = False) -> Epd:
+        ...
+
+    def create(self, epd: Epd, with_response: bool = False) -> Epd | tuple[Epd, Response]:
+        """
+        Create an EPD.
+
+        :param epd: EPD
+        :param with_response: return the response object together with the EPD
+        :return: EPD or EPD with HTTP Response object depending on parameter
+        """
+        response = self._client.do_request(
+            "post",
+            "/epds",
+            json=epd.to_serializable(exclude_unset=True, exclude_defaults=True, by_alias=True),
+        )
+        content = response.json()
+        if with_response:
+            return Epd.parse_obj(content), response
+        return Epd.parse_obj(content)
+
+    @overload
+    def edit(self, epd: Epd, with_response: Literal[True]) -> tuple[Epd, Response]:
```

Comment:
Just wondering, this would work in python 
`edit(epd, with_response=1)` but not with this typing, right?

Reply:
Yes